modules/understanding-upgrade-channels.adoc

```diff
@@ -14,10 +14,112 @@ recommending the appropriate upgrade versions to your cluster. Upgrade channels
 separate upgrade strategies and also are used to control the cadence of updates.
 Channels are tied to a minor version of {product-title}. For instance,
 {product-title} {product-version} channels will never include an upgrade to a
-4.3 release. This ensures administrators make an explicit decision to upgrade to
+4.5 release. This ensures administrators make an explicit decision to upgrade to
 the next minor version of {product-title}. Channels only control updates and
 have no impact on the version of the cluster you install; the
 `openshift-install` binary for a given patch level of {product-title} always
 installs that patch level.
 
-See link:https://access.redhat.com/articles/4495171[OpenShift 4.2 Upgrades phased roll out] for more information on the types of updates and upgrade channels.
+{product-title} {product-version}, which includes the upgrade from the previous 4.3
+release, has three upgrade channels to choose from:
```

Comment:
This "which includes the upgrade from the previous..." bit is new since [the KB article][1].  I'd rather just say:

```
{product-title} {product-version} has the following channels:
```

The fact that those channels may also include y-1 releases to allow cross-minor updates can be discussed later on.

[1]: https://access.redhat.com/articles/4495171

Reply:
It's not new. The only change I made from the article was to remove the "OCP 4.1 included two upgrade channels..." stuff. The above comes right after it and I left it "as is",  other than version, in keeping with our going in position of no content changes. But I thought the 4.1 topic was no longer necessary.

Reply:
> It's not new.

Grr, I just can't read.  You're right; this is fine; carry on.